projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+.retry:
+  retry_options:
+    max: 1
+    when:
+      - always
```

Comment:
same snippet duplicates .plugin-common.yml

Reply:
this is work in progress I will be refactoring the script a lot

Reply:
@ivakoleva currently the installation is failing on random occasions and I am using this pr to find some pattern